test/java/seedu/addressbook/commands/RepeatCommandTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.parser.Parser;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+public class RepeatCommandTest {
+    private Parser parser;
+
+    @Before
+    public void setUp() {
+        parser = new Parser();
+    }
+
+    @Test
+    public void repeatCommand_invalidCommand_throwsException() {
```

Comment:
Good to include negative test case.

Reply:
Perhaps you can create another negative test case for when the previous command is invalid to make your test more comprehensive.